### DISCUSSION TITLE
Prisma Mongodb error

### DISCUSSION THREAD

**Discussion by 05priyanshu** [VaibhavArora314/StyleShare#106] [Q&A] [answered]
I am trying to setup the project locally and the env variables I am using are set as below:

DATABASE_URL="mongodb://localhost:27017/styleshare"
JWT_SECRET="secret"
PORT=3001

On signing up I get this error

![WhatsApp Image 2024-06-04 at 12 17 20_457ce1a3](https://github.com/VaibhavArora314/StyleShare/assets/118009761/2ef6fa20-8109-40ab-9f11-9b82d0ab9716)

**Comment by Ultimateutkarsh11** [comment, marked as answer]
Hey, @05priyanshu Prisma may give an error for the MongoDB replica set, in such case use Mongodb atlas for the database instead of the local database or start a Mongo docker container with the replica set.
